Jenkinsfile

```diff
@@ -0,0 +1,130 @@
+library 'magic-butler-catalogue'
+
+def projectName = "eslint-config"
```

Comment:
You should re-name the repo to be `eslint-config-logdna` before it is too late. and then this should be `eslint-config-logdna`

Reply:
yeah, I just thought we were getting rid of the redundancy naming, but I get that this is a convention more than anything.  I can rename it.

Reply:
with the latest changes this should be `eslint-config-logdna`

Reply:
yep, thanks I fixed it.